packages/workflows/src/handlers/inventory/detach-inventory-items.ts

```diff
@@ -22,10 +22,10 @@ export async function detachInventoryItems({
   }
 
   await Promise.all(
-    data.inventoryItems.map(async ({ variant, inventoryItem }) => {
+    data.inventoryItems.map(async ({ tag, inventoryItem }) => {
       return await productVariantInventoryService.detachInventoryItem(
```

Comment:
I believe here could be a good opportunity to use the remoteLink (available on the container from the context) to dettach the link.

```
await remoteLink.dismiss({ // one (object) or many (array)
  [Modules.PRODUCT]: {
    variant_id: "var_abc",
  },
  [Modules.INVENTORY]: {
    inventory_item_id: "iitem_abc",
  },
});
```

Reply:
The attach has more business logic to be transformed into workflow steps, but this one is straightforward.